Vulkan RenderingSystem needs to be statically linked to glslang and spirv libraries
--- a/CMake/Packages/FindVulkan.cmake
+++ b/CMake/Packages/FindVulkan.cmake
@@ -99,9 +99,25 @@
 			"${OGRE_VULKAN_SDK}/${VK_ARCH}/lib"
 			"$ENV{VULKAN_SDK}/lib"
 		)
+
+  use_pkgconfig(glslang_PKGC glslang)
+  if (NOT glslang_PKGC_FOUND)
+    message(STATUS "glslang not found. Disabling Vulkan support")
+    set(Vulkan_FOUND FALSE)
+  else()
+    set(glslang_LIBRARIES ${glslang_PKGC_LIBRARIES})
+  endif()
+
+  use_pkgconfig(spirv_PKGC spirv)
+  if (NOT spirv_PKGC_FOUND)
+    message(STATUS "spirv not found. Disabling Vulkan support")
+    set(Vulkan_FOUND FALSE)
+  else()
+    set(spirv_LIBRARIES ${spirv_PKGC_LIBRARIES})
+  endif()
 endif()
 
-set(Vulkan_LIBRARIES ${Vulkan_LIBRARY})
+set(Vulkan_LIBRARIES ${Vulkan_LIBRARY} ${glslang_LIBRARIES} ${spirv_LIBRARIES})
 set(Vulkan_INCLUDE_DIRS ${Vulkan_INCLUDE_DIR})
 
 include(FindPackageHandleStandardArgs)
